bricklinkuser/store.go

```diff
@@ -0,0 +1,145 @@
+package bricklinkuser
+
+import (
+	"encoding/json"
+	"io"
+	"net/url"
+)
+
+// CartItemSimple is a simplified representation of an item used to add the item to a cart
+type CartItemSimple struct {
```

Comment:
How did you arrive at this simplified struct? What is `sourceType`?

Reply:
This is what the web UI passes in when you add an item to the cart. Sorry, I don’t know what it means, just found what it was passing in.